java/arcs/android/storage/service/IStorageServiceNg.aidl

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.storage.service;
+
+import arcs.android.storage.service.IMessageCallback;
+import arcs.android.storage.service.IStorageChannel;
+import arcs.android.storage.service.IStorageChannelCallback;
+
+/**
+ * Responsible for opening up communication channels to/from the storage service for proxy messages.
+ */
+// TODO(b/162747024): Rename to IStorageService and replace the existing IStorageService
+interface IStorageServiceNg {
```

Comment:
Do you think it's worth merging this interface with `IMuxedStorageService`? It could be one interface with two methods `openStorageChannel` and `openMuxedStorageChannel`? Or is it better to keep them separate?

Reply:
I think we should keep them separate.

I was planning to make a separate "muxed" version of the AndroidStorageServiceEndpointManager and AndroidServiceEndpoint. If this is the approach we go for then I think it makes sense to keep these interfaces separate because we wouldn't want the EndpointManager to have the ability to open a muxed storage channel or to let the MuxedEndpointManager to have the ability to open a non-muxed storage channel.

Reply:
Sounds good!